### PR TITLE
Allow to display wallet ballance

### PR DESCRIPTION
**Pull request by nocmar**

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,46 @@
 import React from 'react'
 
-import {WalletProvider, Wallet } from 'starterra-tool-dapp'
+import { WalletProvider, Wallet, Tokens } from 'starterra-tool-dapp'
 import { avaliableNetworks } from './avaliableNetworks'
 import 'starterra-tool-dapp/dist/index.css'
 
+
+const tokens:Tokens=
```
Comment (Mskalba) [inline example/src/App.tsx:8]: Maybe lets not change it now, but I would love to work with typescript the way:

type TTokens

interface ITokens

So naming convention for interfaces and types with prefix "T" / "I"

File: src/graphql/balanceQuery.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { gql } from '@apollo/client'
+
+interface Item {
+  token: string
+  walletAddress: string
+}
+
+const balanceItem = ({ token, walletAddress }: Item) => `
```
Comment (Mskalba) [inline src/graphql/balanceQuery.ts:8]: const balance = await terra.wasm.contractQuery(token, {balance: {address: walletAddress}});

Maybe we do not need mantle at all here.

File: src/graphql/useBankBalance.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { gql, useQuery, ApolloError } from '@apollo/client'
+import { TokenBalance, Tokens } from '../types/token'
+
+export interface TokenBalanceResponse {
+  Denom: string
+  Amount: string
+}
+
+const QUERY = gql`
+  query BankBalancesAddress($address: String) {
```
Comment (Mskalba) [inline src/graphql/useBankBalance.ts:10]: const terra = new LCDClient({
    URL: 'https://fcd.terra.dev',
    chainID: 'columbus-4'
});

const balance = await terra.bank.balance(accounts[0]);

**Review by Mskalba — APPROVED** [review]

File: src/graphql/balanceQuery.ts (hunk 2 above)
Comment (nocmar) [reply src/graphql/balanceQuery.ts]: ok I updated it to use terra.js

File: src/graphql/useBankBalance.ts (hunk 3 above)
Comment (nocmar) [reply src/graphql/useBankBalance.ts]: same there I updated it to use terra.js

File: example/src/App.tsx (hunk 1 above)
Comment (nocmar) [reply example/src/App.tsx]: ok I will create a new issue and update it all, will be great to setup linter for that

**Review by Mskalba — APPROVED** [review]

**Pull request merged**